lib/graphql/queries.js

```diff
@@ -77,7 +77,10 @@ export async function fetchInvoiceByDateRange(
           createdAt
           description
           hostCurrencyFxRate
-          invoiceTemplate
+          ... on Credit {
```

Comment:
Why only on credit and not debit? I don't get the rationale

Reply:
Ops, you are correct. We need to add `Debit` as well and I've done that. 👍🏽